gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -101,14 +97,6 @@
     }
   }
 
-  public static int findAvailablePort(int defaultPort) {
-    try (ServerSocket tempSocket = new ServerSocket(0)) {
-      return tempSocket.getLocalPort();
-    } catch (IOException e) {
-      return defaultPort;
-    }
-  }
-
   private static Path installedGcdPath() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method installedGcdPath() has an NPath complexity of 420](https://www.codacy.com/app/mziccard/gcloud-java/file/2329630392/issues/source?bid=3123103&fileBranchId=3205624#l100)